pwd-service/src/main/java/org/pwd/web/download/DownloadController.java

```diff
@@ -1,10 +1,13 @@
 package org.pwd.web.download;
 
-import org.pwd.domain.contact.PlainTextEmailMessage;
+import com.lyncode.jtwig.JtwigModelMap;
+import com.lyncode.jtwig.JtwigTemplate;
+import com.lyncode.jtwig.configuration.JtwigConfiguration;
+import com.lyncode.jtwig.resource.ClasspathJtwigResource;
+import org.pwd.domain.contact.HtmlEmailMessage;
```

Comment:
zrob w tej klasie metode /download/HASH
hasha porownaj z sha-1(nazwatemplaty+nazwacms+dzisiejszadata) jak sie zgadza przeslij plik do pobrania jesli nie to to stronke z bledem, ze link juz wygasl ;)